installer/templates/new/web/views/error_helpers.ex

```diff
@@ -8,9 +8,11 @@ defmodule <%= application_module %>.ErrorHelpers do
   @doc """
   Generates tag for inlined form input errors.
   """
-  def error_tag(form, field) do
+  def error_tag(form, field, opts \\ []) do
+    opts = Keyword.put_new(opts, :class, "help-block")
```

Comment:
This does seem to make more sense when there's only one possible option, I'll update it